crowbar_framework/lib/crowbar/checks/network.rb

```diff
@@ -67,19 +67,11 @@ def ip_configured?
       end
 
       def firewall_disabled?
-        if system("sudo LANG=C iptables -n -L | grep -qvE '^$|^Chain [^ ]|^target     prot'")
-          return false
-        end
-
-        true
+        return ! system("sudo LANG=C iptables -n -L | grep -qvE '^$|^Chain [^ ]|^target     prot'")
```

Comment:
Style/RedundantReturn: Redundant `return` detected. (https://github.com/bbatsov/ruby-style-guide#no-explicit-return)<br>Style/SpaceAfterNot: Do not leave space between `!` and its argument. (https://github.com/bbatsov/ruby-style-guide#no-space-bang)